datasets/gov/aaa/pakuociu_tiekimas.csv

```diff
@@ -0,0 +1,20 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/aaa/pakuociu_tiekimas,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,PakuociuTiekimasVidausRinkai,,,,,,2,,,Pakuočių tiekimo vidaus rinkai apskaitos duomenys,
+,,,,,metai,integer,,,,2,open,,Metai,Ataskaitiniai metai
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,medziaga,string,,,,4,open,,Pakuotės medžiagos pavadinimas,"Pakuočių ir pakuočių atliekų tvarkymo taisyklės, patvirtintos Lietuvos Respublikos aplinkos ministro 2002-06-27 įsakymu Nr. 348, Mokesčio už aplinkos teršimą deklaracijų FR0521, FR0522, FR0523 ir FR0524 formų ir jų pildymo taisyklės, patvirtintos Valstybinės mokesčių inspekcijos prie Lietuvos Respublikos finansų ministerijos viršininko ir Lietuvos Respublikos aplinko ministro 2008 m. gruodžio 8 d. įsakymu Nr. VA-61/D1-658"
+,,,,,,enum,,,"""Stiklinė""",,,,Stiklinė pakuotė,
+,,,,,,,,,"""Plastikinė""",,,,Plastikinė pakuotė,
+,,,,,,,,,"""Popierinė-kartoninė""",,,,Popierinė ir kartoninė pakuotė,
+,,,,,,,,,"""Metalinės (iš viso)""",,,,Metalinės pakuotės (iš viso),
+,,,,,,,,,"""juodųjų metalų""",,,,Juodųjų metalų pakuotė,"Duomenys apskaičiuoti vertinimo būdu, pagal atliekų tvarkytojų teikiamų ataskaitų duomenis (Atliekų tvarkymo ir susidarymo apskaitos ir ataskaitų teikimo taisyklės). Nuo 2021 m. pagal Gamintojų ir importuotojų, tiekiančių šalies rinkai supakuotus gaminius, metinių ataskaitų duomenis."
+,,,,,,,,,"""aliuminė""",,,,Aliuminio pakuotė,"Duomenys apskaičiuoti vertinimo būdu, pagal atliekų tvarkytojų teikiamų ataskaitų duomenis (Atliekų tvarkymo ir susidarymo apskaitos ir ataskaitų teikimo taisyklės). Nuo 2021 m. pagal Gamintojų ir importuotojų, tiekiančių šalies rinkai supakuotus gaminius, metinių ataskaitų duomenis."
+,,,,,,,,,"""Medinė""",,,,Medinė pakuotė,
+,,,,,,,,,"""Kombinuota (iš viso)""",,,,Kombinuota pakuotė (iš viso),
+,,,,,,,,,"""kombinuota popierinė""",,,,kombinuota popierinė pakuotė,
+,,,,,,,,,"""kombinuota kita""",,,,kombinuota kita,
+,,,,,,,,,"""Kita""",,,,Kita,
+,,,,,,,,,"""Visos medžiagos""",,,,Visos medžiagos (bendras kiekis),
+,,,,,kiekis,number,t,,,4,open,,"Patiekta vidaus rinkai","Duomenys parengti pagal Gamintojų ir importuotojų, tiekiančių šalies rinkai supakuotus gaminius, metinių ataskaitų duomenis ir Mokesčio už aplinkos teršimą pakuočių atliekomis aktualių deklaracijų duomenis."
```

Comment:
Prašau pakartotinai įvertinti, ar nurodytas duomenis atitinkantis duomenų tipas?
Atkreipiu dėmesį, kad duomenys pateikti be kablelio.
![image](https://user-images.githubusercontent.com/85924252/218459827-7aba1b72-50b0-4a70-88c5-8cb4ab98dbbd.png)

Plačiau apie duomenų tipus: https://atviriduomenys.readthedocs.io/dsa/duomenu-tipai.html

Reply:
Duomenų tipas pakeistas iš number į integer.